addon/lib/auth-request.js

```diff
@@ -19,36 +20,26 @@ function pingServer(config, title, data, addon) {
       }
     });
 
-    getMe(headers, config.BASE_URL, function(response) {
-      let id = '';
-      if (response.json) {
-        id = response.json.id;
-      }
-
-      request({
-        url: config.BASE_URL + '/api/metrics/?format=json',
-        content: JSON.stringify(formatEvent(config.TESTPILOT_PREFIX, title, id, data, addon)),
-        headers: headers,
-        onComplete: function(resp) { // eslint-disable-line no-unused-vars
-          // console.error(resp);
-        }
-      }).post();
-    });
+    cb(headers);
   });
 }
 
-function getMe(headers, BASE_URL, cb) {
-  request({
-    url: BASE_URL + '/api/me?format=json',
-    headers: headers,
-    onComplete: cb
-  }).get();
+function sendMetric(config, title, data, addon) {
+  authRequest(config, function(headers) {
+    request({
+      url: config.BASE_URL + '/api/metrics/?format=json',
+      content: JSON.stringify(formatEvent(config.TESTPILOT_PREFIX, title, data, addon)),
+      headers: headers,
+      onComplete: function(resp) { // eslint-disable-line no-unused-vars
+        // console.error(resp);
+      }
+    }).post();
+  });
 }
 
-function formatEvent(PREFIX, title, id, data, addon) {
+function formatEvent(PREFIX, title, data, addon) {
```

Comment:
Hmm, though yeah we probably don't need the user ID here, since we could just pick that up on the server side if we really wanted it from the authenticated user.